docs/release-notes/cloud/cloud-2022-03.md

```diff
@@ -0,0 +1,113 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud March 2022.
+included_jira_versions: ['2022.Q1.5', '2022.Q1.6', '2022.Q2.1']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/5.2.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.6.3
+---
+
+# Cloud March 2022
+
+These release notes are for the Codacy Cloud updates during March 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   GitLab project members with the Maintainer role can now perform the following operations on Codacy:
+
+    -   Adding the project
+    -   Configuring the project
+    -   Changing the following analysis settings: ignoring issues and files, configuring code patterns, configuring file extensions, and managing branches
+
+    [See the updated permissions](../../organizations/roles-and-permissions-for-synced-organizations.md) for all GitLab roles. (CY-5876)
+
+-   Included ESLint 8 as a new supported tool and deprecated ESLint 7.
+
+    New repositories will use ESLint 7 by default, but **Codacy won't provide more updates for ESLint 7 and will remove ESLint 7 on April 4, 2023**. (CY-5848)
```

Comment:
Think there's a small snafu here with version numbers. New default version for ESLint is `8` but on this sentence `New repositories will use ESLint 7 by default, (...)` it's stated otherwise.

Reply:
Yes, this is a mistake. I'm going to fix it now.